analyses/cell-type-wilms-tumor-06/notebook_template/03_clustering_exploration.Rmd

```diff
@@ -0,0 +1,618 @@
+DT::datatable(top5[, c(1, 9, 11, 12)], caption = ("top 5 marker genes"), 
+              extensions = 'Buttons', 
+              options = list(  dom = 'Bfrtip',
+                               buttons = c( 'csv', 'excel')))
+```
+
+
+
+### Enrichment analysis of marker genes
+
+Here we perform enrichment analysis of the marker genes found in the previous section for each Seurat cluster.
+
+We defined as universe/background all the genes expressed in the dataset, meaning the rownames of the Seurat object.
+
+We used three gene sets from MSiGDB:
+
+- the hallmark gene sets  are coherently expressed signatures derived by aggregating many MSigDB gene sets to represent well-defined biological states or processes.
+- the C3 : regulatory target gene sets  based on gene target predictions for microRNA seed sequences and predicted transcription factor binding sites.
+- the C8 : cell type signature gene sets  curated from cluster markers identified in single-cell sequencing studies of human tissue.
+
+We used enricher function from clusterProfiler to perform enrichment analysis.
+
+```{r enrichment}
+# define background genes = universe for enrichment
+background <-  rownames(srat)
+# Define gene signature per cluster
+signatures <- list()
+for( i in unique(s.markers$group1)){
+  signatures[[paste0("cluster ",i)]] <- s.markers$feature[s.markers$group1 == i]
+}
+```
+
+#### Hallmarks MSiGDB gene sets
+
+```{r fig.width=15, fig.height=8, out.width='100%'}
+
+    Enrichment_plot(category = "H", signatures = signatures, background = background)
+ 
+```
+
+
+EMT signature should be enriched in stroma cluster.
+E2F/proliferation should be enriched in blastema cluster.
+MYC(N), TP53 must be enriched in blastema cluster.
+
+
+
+#### C8 MSiGDB cell type signature gene sets
+
+```{r fig.width=20, fig.height=25, out.width='100%'}
+Enrichment_plot(category = "C8", signatures = signatures, background = background)
+```
+
+The MSigDB C8 gene set is quite relevant for kidney and nephroblastoma annotations. 
+Epithelial (cancer and normal) cells should be enriched in mature/adulte kidney pathways while blastema cancer cells will show enrichment of fetal kidney development pathway / cap mesenchyme. 
+
+### Cell annotation
+
+#### Annotation from SingleR (no cancer or kidney specific dataset)
+
+Here, we quickly checked annotations that are present in the _processed rds object. 
+However, the automated annotation have not been performed using a cancer specific reference or a kidney reference. 
+We do not expect a nice labelling of the cells as the overlap of cell types between the reference and the query dataset is poor. 
+This support the need to do a proper label transfer from the fetal kidney atlas, which is imho the best reference that can be applied to a Wilms tumor query. 
+
+
+```{r fig.width=15, fig.height=10, out.width='100%'}
```

Comment:
The title doesn't fully fit here, so can you make the `fig.width=20` instead of 15 for **all chunks** (here and below) that use the `visualize_metadata()` function?